bitnami/memcached/README.md

```diff
@@ -49,6 +49,7 @@ The following tables lists the configurable parameters of the Memcached chart an
 
 | Parameter                                | Description                                                                                            | Default                                                      |
 |------------------------------------------|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
+| `architecture`                           | Memcahed architecture. Allowed values: standalone or high-availability                                 | `standalone`                                                 |
```

Comment:
Please put this line after `clusterDomain` that is the order it has in the `values.yaml`

Reply:
Done